airflow/www/decorators.py

```diff
@@ -101,8 +101,10 @@ def has_dag_access(**dag_kwargs):
     def decorator(f):
         @functools.wraps(f)
         def wrapper(self, *args, **kwargs):
+            request_arguments = request.args.to_dict()
+            request_arguments.update(request.form.to_dict())
```

Comment:
The decorator has been used in many people. Does it work for GET/PUT endpoint as well?

Reply:
Yup, request.args will take in URL parameters from GET, POST, and PUT. request.form takes in the form parameters. At least on the environment we're running on we confirmed that this wrapper works for both GET endpoints and the POST endpoint with form data.